src/components/calendar/calendar.js

```diff
@@ -920,6 +924,8 @@ export const BCalendar = Vue.extend({
           attrs: {
             title: label || null,
             type: 'button',
+            disabled: btnDisabled,
```

Comment:
We use `aria-disabled` instead of the `disabled` attribute for accessibility reasons.